index.js

```diff
@@ -237,7 +237,7 @@ module.exports = function (content) {
             return result.css.toString();
         } catch (err) {
             formatSassError(err);
-            err.file && this.dependency(err.file);
+            err.file && addNormalizedDependency(err.file);
```

Comment:
I don't think that this is necessary. `formatSassError` [is already normalizing it](https://github.com/strayiker/sass-loader/blob/c51fff0c25d633b4df010e6ba3a95027d43c04ad/index.js#L55).